superset-frontend/src/explore/reducers/exploreReducer.js

```diff
@@ -27,6 +27,11 @@ import * as actions from '../actions/exploreActions';
 
 export default function exploreReducer(state = {}, action) {
   const actionHandlers = {
+    [actions.VIEW_IN_SQLLAB]() {
+      return {
+        ...state,
+      };
+    },
```

Comment:
You can omit this, since it's a no-op. If you look at the bottom of the `exploreReducer.js` file you can see that when an action doesn't exist it simply returns the unmodified state.